user/models.py

```diff
@@ -1,8 +1,14 @@
+import os
+
 from django.db import models
 from django.contrib.auth.models import AbstractUser
 from django.utils.text import slugify
 
 
+def get_image_path(instance, filename):
```

Comment:
Add doc string here, describe instance type, filename type and type of return value + short function description